app/src/components/widgets/multiLevelWidgets/cumulativeTrendChart/chartjsConfig.js

```diff
@@ -40,9 +53,11 @@ const convertIntoPercents = (datasets) => {
       },
       dataset,
       {
-        data: dataset.data.map(
-          (value, index) => -((-value / totalDataset.data[index]) * 100).toFixed(2),
-        ),
+        data: dataset.data.map((value, index) => {
+          const totalValue = getTotalValue(totalDataset, datasets, dataset.label, index);
+
+          return -((-value / totalValue) * 100).toFixed(2);
```

Comment:
This expression repeates two times (here and [here](https://github.com/reportportal/service-ui/pull/2527/files#diff-52b27d49d71aaf8a3c5952f64e88efed18721548e348df25167e2b52b70c21b3R184)), so I think it can be moved to the `getTotalValue` function.
What do you think?